setup.py

```diff
@@ -35,7 +35,7 @@
 extras_require = {
     'voice': ['PyNaCl>=1.3.0,<1.6'],
     'docs': [
-        'sphinx==4.4.0',
+        'sphinx==7.2.6',
```

Comment:
I want to note that Sphinx 7.2.6 does not support Python 3.8 ([changelog](https://www.sphinx-doc.org/en/master/changes.html#id32), [pr](https://github.com/sphinx-doc/sphinx/pull/11511)) since they dropped support for 3.8 in Sphinx 7.2.0. The only latest version that does support 3.8 is Sphinx 7.1.2

Other than that everything seems fine

Reply:
If that's the case, I can change the version to 7.1.2.  The patch will still work.

Reply:
Sounds good